def/taskUtils.go

```diff
@@ -105,7 +108,8 @@ func clusterSizeAvgMMMPU(tasks []Task, taskObservation func(task Task) []float64
 			} else {
 				// skip this value
 				// there is an error in the task config.
-				log.Println(err)
+				elektronLogging.ElektronLog.Log(elekLogT.ERROR, log.ErrorLevel,
```

Comment:
Replace `elekLogT.ERROR` to `elekLogT.CONSOLE`.

Reply:
Done